lecture_5/homework/app/models/warrior.rb

```diff
@@ -14,4 +14,10 @@ class Warrior < ApplicationRecord
 
   scope :alive, -> { where('death_date IS NULL') }
   scope :dead, -> { where('death_date IS NOT NULL') }
+
+  after_save :update_siege_abilities
```

Comment:
Jeśli usuniemy warriora z budynku, to w budynku siege ability nie zaktualizuje sie :(